pywemo/ouimeaux_device/dimmer.py

```diff
@@ -0,0 +1,39 @@
+from . import Device
+
+
+class Dimmer(Device):
```

Comment:
You can inherit from Switch - and then remove `set_state`, `on`, `off` and `toggle` methods.

To do this import `Switch` rather than `Device`.
`from .switch import Switch`

And then change this line to
`class Dimmer(Switch):`